ftw/subsite/browser/ressources/integration.ftw-mobile.theme.scss

```diff
@@ -17,7 +17,7 @@
 #portal-languageselector-wrapper #portal-languageselector{
   display: none;
 
-  @include tablet {
+  @include screen-medium {
```

Comment:
Where is `screen-medium` defined?
Not in https://github.com/4teamwork/ftw.theming/pull/85 🤔

Reply:
The mixin is defined here: https://github.com/4teamwork/ftw.theming/blob/master/ftw/theming/resources/scss/globals/variables.scss#L211